src/Ocelot/LoadBalancer/LoadBalancers/CookieStickySessions.cs

```diff
@@ -17,27 +13,27 @@ public class CookieStickySessions : ILoadBalancer
         private readonly int _keyExpiryInMs;
         private readonly string _key;
         private readonly ILoadBalancer _loadBalancer;
-        private readonly ConcurrentDictionary<string, StickySession> _stored;
+        private readonly IStickySessionStorage _storage;
         private readonly IBus<StickySession> _bus;
         private readonly object _lock = new();
 
-        public CookieStickySessions(ILoadBalancer loadBalancer, string key, int keyExpiryInMs, IBus<StickySession> bus)
+        public CookieStickySessions(ILoadBalancer loadBalancer, string key, int keyExpiryInMs, IBus<StickySession> bus, IStickySessionStorage storage)
```

Comment:
> substitute CookieStickySessionsCreator to inject into CookieStickySessions own storage mechanism

Because new added interface object can be injected by this creator, we need to explain to developer how to implement custom creator using `ILoadBalancerCreator` interface.
**So, new paragraph in docs is required.**